patterns/atoms/checkbox/index.tsx

```diff
@@ -0,0 +1,89 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import colors from '../colors';
+import Copy from '../copy';
+
+/**
+ * @name Checkbox
+ */
+
+export interface CheckboxProps {
+	/** @name Label Text @default Lorem Ipsum */ labelText?: string;
+	/** @name Name @default checkbox */ name?: string;
+	/** @name Value @default value */ value?: string;
```

Comment:
hide `name` and `value` properties in Alva

Reply:
done